variants/README.md

```diff
@@ -446,6 +433,20 @@ It supported self-hosted clusters.
 
 This variant was compatible with Kubernetes 1.27, 1.28, 1.29, and 1.30 clusters.
 
+### metal-k8s-1.28: Metal Kubernetes 1.28 node
+
+The [metal-k8s-1.28](metal-k8s-1.28/Cargo.toml) variant includes the packages needed to run a Kubernetes node on bare metal.
+It supports self-hosted clusters.
+
+This variant is compatible with Kubernetes 1.28, 1.29, 1.30, and 1.31 clusters.
+
+### metal-k8s-1.29: Metal Kubernetes 1.29 node
+
+The [metal-k8s-1.29](metal-k8s-1.29/Cargo.toml) variant includes the packages needed to run a Kubernetes node on bare metal.
+It supports self-hosted clusters.
+
+This variant is compatible with Kubernetes 1.29, 1.30, 1.31, and 1.32 clusters.
+
```

Comment:
Should use past tense for these now that they are deprecated

Reply:
Nice catch!